vpr/src/server/gateio.h

```diff
@@ -26,16 +26,16 @@ namespace server {
  * 
  * Operable only with a single client. As soon as client connection is detected
  * it begins listening on the specified port number for incoming client requests,
- * collects and encapsulates them into tasks.
- * The incoming tasks are extracted and handled by the top-level logic (TaskResolver). 
- * Once the tasks are resolved by the TaskResolver, they are returned 
- * to be sent back to the client as a response.
+ * collects and encapsulates them into tasks (see @ref Task).
+ * The incoming tasks are extracted and handled by the top-level logic @ref TaskResolver in main thread.
```

Comment:
in *the* main thread

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/adaa276ef2e5b4669a2c144bf7cc1b0fd25a47ce
(will be updated later)